types/amcharts/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for amCharts
```

Comment:
This line
```
declare namespace AmCharts {
```

should add export in front so it can be imported

```
export declare namespace AmCharts {
```

Reply:
```typescript
export declare
```
is redundant, I changed to

```typescript
export
```